tests/rest/client/v1/test_login.py

```diff
@@ -522,7 +522,9 @@ def test_multi_sso_redirect_to_cas(self):
             shorthand=False,
         )
         self.assertEqual(channel.code, 302, channel.result)
-        cas_uri = channel.headers.getRawHeaders("Location")[0]
+        location_headers = channel.headers.getRawHeaders("Location")
+        assert location_headers
```

Comment:
Weirdly it seems that mypy doesn't understand `self.assertTrue(location_headers)`. 🤷 Not sure there's something better we can do for this (and similar instances).

Reply:
Does it understand `self.assertIsNotNone(location_headers)` instead?

Reply:
I tried a handful of variations of this (`assertTrue`, `assertIsInstance`, `assertIsNotNone`, etc.) I think mypy just doesn't know what these are. The only references I can find to them in mypy are in the stubs file.

I was wondering if this is because were using the twisted unit test classes (although this does eventually inherit from the Python unit testing framework). 🤷

Reply:
This seems to be python/mypy#5088, which points to python/mypy#4063.

Reply:
I see. Oh well, `assert` it is for now then.